frontend/src/App.scss

```diff
@@ -7,6 +7,18 @@ ul.pagination li {
   cursor: pointer;
 }
 
+.modal-dialog.modal-xl {
```

Comment:
provide some alternatives to the standard modal size

Reply:
Nice addition!